app/models/affinity.rb

```diff
@@ -1,6 +1,8 @@
 class Affinity < AffinityBase
   include Garden::Fruit
 
+  scope :by_kind, -> (code){ where(affinity_kind_id: AffinityKind.find_by_code(code.to_sym).id) }
```

Comment:
aca podes probar si funciona asi :+1: 
scope :by_kind, -> (code){ where(affinity_kind: code) }

Reply:
Me da error de esa manera:
`     Failure/Error: scope :by_kind, -> (code){ where(affinity_kind: code) }
     
     NoMethodError:
       undefined method `arel_table' for AffinityKind:Class`

Reply:
scope :by_kind, -> (code){ where(affinity_kind_id: code.id) }

y las llamadas en vez de pasar el simbolo :same_person, pasale AffinityKind.same_person